xds/internal/balancer/clusterresolver/configbuilder.go

```diff
@@ -284,55 +250,12 @@ func dedupSortedIntSlice(a []int) []int {
 	return a[:i+1]
 }
 
-// rrBalancerConfig is a const roundrobin config, used as child of
-// weighted-roundrobin. To avoid allocating memory everytime.
-var rrBalancerConfig = &internalserviceconfig.BalancerConfig{Name: roundrobin.Name}
-
 // priorityLocalitiesToClusterImpl takes a list of localities (with the same
 // priority), and generates a cluster impl policy config, and a list of
-// addresses.
+// addresses with their sets their path hierarchy set to [priority-name,
+// locality-name], so priority and the xDS LB Policy know which child policy
+// each address is for.
```

Comment:
There seems to be some grammatical error in this comment. Please fix.

Reply:
Oh, yikes haha. Switched to "addresses with their path hierarchy set to [priority-name, locality-name]"